### ISSUE TITLE
twisted.conch.ssh.filetransfer should be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8675]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8675|
|Type|enhancement|
|Created|2016-07-25 13:25:48Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8675 8675
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469453148603310 1469453148603310
changetime__1469498204064675 1469498204064675
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)<a name="note_1"></a>|@einarfd removed owner|
|-|-|

Added a merge request that fixes this at https://github.com/twisted/twisted/pull/411

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 2d4b239e897181d085b3c155421042ad05f83529
```
#!CommitTicketReference repository="" revision="2d4b239e897181d085b3c155421042ad05f83529"
Merge pull request #414 from twisted/8675-einarfd-ssh-filetransfer-2

Author: einarfd
Reviewer: rodrigc
Fixes: #8675

Port twisted.conch.ssh.filetranfer to Python 3
```
